core/iwasm/common/wasm_suspend_flags.h

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (C) 2023 Amazon Inc.  All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ */
+
+#ifndef _WASM_SUSPEND_FLAGS_H
+#define _WASM_SUSPEND_FLAGS_H
+
+#include "bh_platform.h"
+#include "gnuc.h"
```

Comment:
Could you include "gnuc.h" in "bh_platform.h" instead? We often only include "bh_platfrom.h" to include all the required APIs in `core/shared/platform` and `core/shared/utils`.

Reply:
done